src/components/v2/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Toolbar from '@mui/material/Toolbar';
+import { NavLink } from 'react-router-dom';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import Drawer from '@mui/material/Drawer';
+import { uid } from 'react-uid';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material';
+import { ReactComponent as XVSIcon } from '../../../assets/img/xvs.svg';
+import { ReactComponent as Logo } from './assets/logo.svg';
+
+const menuItems = [
+  {
+    href: '/dashboard',
+    text: 'Dashboard',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/vote',
+    text: 'Vote',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/xvs',
+    text: 'XVS',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/market',
+    text: 'Market',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/vault',
+    text: 'Vault',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/transaction',
+    text: 'Transaction history',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/dev',
+    text: 'Dev',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+];
+
+interface IProps {
+  drawerWidth: number;
+}
+
+export const Sidebar = ({ drawerWidth }: IProps) => {
```

Comment:
If the drawerWidth is part of the theme we don't need to pass it as a prop

Reply:
okay. gonna define it inside theme scope